### ISSUE TITLE
trigger JS events after any ajax requests that change the form DOM

### ISSUE THREAD

**Issue by pryley** [WordImpress/Give#204]
I have a custom script that styles the radio/checkbox/select fields and converts labels to floating labels (similar to this: http://clubdesign.github.io/floatlabels.js/).

![ccard](https://cloud.githubusercontent.com/assets/134939/9217105/0e1c2150-40f6-11e5-997e-1151970eaffb.png)

I need to be able to trigger a refresh anytime the DOM is changed via ajax. All this requires is something like the following inserted at the end of any successful ajax request: 

`jQuery( document ).trigger( 'give_gateway_loaded', [response] );`

Example: `function give_load_gateway() // give-ajax.js:197`

**Comment by DevinWalker** [comment]
@pryley would you be interested in submitting a pull request for this?

**Comment by pryley** [comment]
@DevinWalker sure.

**Comment by DevinWalker** [comment]
Cool! :+1:

**Comment by pryley** [comment]
@DevinWalker Hey were you referring to the event trigger or the float labels script? I understood the latter from the issue tags so I converted an internal script to a plugin and integrated it into Give #231 .

If not, no worries.
